frontend/dist/assets/tracim/tinymceInit.js

```diff
@@ -73,6 +73,7 @@
             fileTag.id = 'hidden_tinymce_fileinput'
             fileTag.type = 'file'
             $('body').append(fileTag)
+            $(`#${fileTag.id}`).hide()
```

Comment:
It would be better to not have the input getting visible in first place instead of adding some code to hide it afterwards.

If it's really the only way of hiding it, explain why

Reply:
Yes, it's changed